tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1210,13 +1205,24 @@ public function userHasSharedSpace(
 		}
 		$fullUrl = $baseUrl . "ocs/v2.php/apps/files_sharing/api/v1/shares";
 
-		return HttpRequestHelper::post($fullUrl, "", $user, $this->featureContext->getPasswordForUser($user), [], $body);
+		$this->featureContext->setResponse(
+			HttpRequestHelper::post(
+				$fullUrl,
+				"",
+				$user,
+				$this->featureContext->getPasswordForUser($user),
+				[],
+				$body
+			)
+		);
```

Comment:
I took out the early return and saved the response to featureContext.
Then the next HTTP and OCS status code checks work.